Source/WebCore/rendering/svg/RenderSVGImage.cpp

```diff
@@ -77,13 +77,18 @@ SVGImageElement& RenderSVGImage::imageElement() const
     return downcast<SVGImageElement>(RenderSVGModelObject::element());
 }
 
+Ref<SVGImageElement> RenderSVGImage::protectedImageElement() const
+{
+    return imageElement();
+}
+
 FloatRect RenderSVGImage::calculateObjectBoundingBox() const
 {
     LayoutSize intrinsicSize;
     if (CachedImage* cachedImage = imageResource().cachedImage())
         intrinsicSize = cachedImage->imageSizeForRenderer(nullptr, style().effectiveZoom());
 
-    SVGLengthContext lengthContext(&imageElement());
+    SVGLengthContext lengthContext(protectedImageElement().ptr());
```

Comment:
Since `SVGLengthContext` just holds a bare pointer internally, if there is any value to using a `protectedImageElement()` it seems like it should be as a `RefPtr` scoped outside the declaration of `lengthContext`.